about.html

```diff
@@ -12,52 +12,87 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
+  <div class="container">
+  <header>
+    <nav class= "headerBar">
+        <div id="navLogo">
+            <img src="img/lambda-black.png"/>
+        </div>
+        <a href="index.html" class="navButtons">Home</a>
```

Comment:
If you prefer, you can reference the anchor tags without using classes by using nav a .  Either way is fine.